lib/xebow/keys.ex

```diff
@@ -102,20 +119,65 @@ defmodule Xebow.Keys do
     poll_timer_ms = 15
     :timer.send_interval(poll_timer_ms, self(), :update_pin_values)
 
+    animations =
+      Animation.types()
+      |> Enum.map(&Animation.new(type: &1))
+
     {:ok,
      %{
        pins: pins,
        keyboard_state: keyboard_state,
-       hid: hid
+       hid: hid,
+       animations: animations,
+       current_animation_index: 0
```

Comment:
:information_source: (informational, no action required)

Keeping this data in `Keys` feels a little weird long-term, but you also noted that in the PR description. I'm fine with this in this PR. In a future PR we may want to try to work this data into `xebow.ex` and use that for the application logic that glues the other components together.

Reply:
See my possibly related documentation update calling out the fact that `keys.ex` has too many responsibilities: https://github.com/ElixirSeattle/xebow/pull/52/files#diff-adc9367d84d7e3a10f70650eece5eae2R7-R14